src/views/Events/index.js

```diff
@@ -38,44 +40,24 @@ const Events = (props) => {
         allEvents = await gordonEvent.getAllGuestEvents();
       }
       setAllEvents(allEvents);
-
-      // Load filters from UrlParams if they exist
-      if (props.location.search) {
-        const urlParams = new URLSearchParams(props.location.search);
-        let willIncludePast = false;
-        const filtersFromURL = [];
-
-        for (const key of urlParams.keys()) {
-          if (key === 'Past') {
-            willIncludePast = true;
-          } else {
-            filtersFromURL.push(key);
-          }
-        }
-
-        setFilters(filtersFromURL);
-        setIncludePast(willIncludePast);
-        setOpen(willIncludePast || filtersFromURL.length > 0);
-      }
-
       setLoading(false);
     };
+    loadEventsPage();
+  }, [props.authentication]);
 
-    loadEvents();
-  }, [props.authentication, props.location.search]);
-
+  // When the actual events data changes update the events
```

Comment:
I'm not sure what this comment means. What is "the actual events data"? Also, I think this `useEffect` is small enough that it speaks for itself.